[DOCS] Create robots.txt / Avoid search results showing docs of old Solidity versions (#10972)
ethereum/solidity: ReleaseChecklist.md

 - [ ] Wait for the ``~ethereum/ubuntu/ethereum-static`` PPA build to be finished and published for *all platforms*. SERIOUSLY: DO NOT PROCEED EARLIER!!! *After* the static builds are *published*, copy the static package to the ``~ethereum/ubuntu/ethereum`` PPA for the destination series ``Trusty``, ``Xenial`` and ``Bionic`` while selecting ``Copy existing binaries``.

### Documentation
 - [ ] Build the new version on https://readthedocs.org/projects/solidity/ (select `latest` at the bottom of the page and click `BUILD`).
 - [ ] In the admin panel, select `Versions` in the menu and set the default version to the released one.

>  - [ ] If it is a non-breaking release, add the previous release version to the ``robots.txt`` file.

Reviewer: I'd make it clearer that the purpose of adding it is to prevent it from being indexed:
```suggestion
 - [ ] If it is a non-breaking release, block indexing of previous release version in the ``robots.txt`` file.
```
